packages/sarif-to-comment/test/empty-comment.test.ts

```diff
@@ -0,0 +1,25 @@
+import * as fs from "fs";
+import * as path from "path";
+import * as assert from "assert";
+// transform function
+import { postComment } from "../src/index";
+
+console.log(__dirname);
+const sarifDir = path.join(__dirname);
+const actualFilePath = path.join(sarifDir, "/sarif.json");
+describe("Testing", () => {
+    it(`Testing failure due to empty sarif`, async function () {
+        const actualOptions = {
+            sarifContent: JSON.parse(fs.readFileSync(actualFilePath).toString()),
+            postingURL: "http://localhost",
+            sarifContentOwner: "aa",
+            sarifContentRepo: "aa",
+            sarifContentBranch: "aa",
+            token: "aa"
+        };
+
+        assert.throws(async () => {
```

Comment:
[assert.throws(fn[, error][, message])](https://nodejs.org/api/assert.html#assertthrowsfn-error-message) is for sync functions.

You should use [assert.rejects(asyncFn[, error][, message])](https://nodejs.org/api/assert.html#assertrejectsasyncfn-error-message) instead for `assert.throws`.

Reply:
e.g.
```js
await assert.rejects(
  async () => {
    await postComment(actualOptions);
  },
  {
    name: 'TypeError??', //  error type
    message: 'There are no results in this SARIF run 0, exiting without a comment !' // error message
  }
);
```